smarts/core/agent.py

```diff
@@ -32,6 +33,15 @@
 logger = logging.getLogger(__name__)
 
 
+@dataclass
+class Adapter:
+    space: gym.spaces
```

Comment:
Should the type be `gym.Space` ?

https://github.com/openai/gym/blob/925823661d4bc14145f03d73e8f2888de44e49ae/gym/spaces/space.py#L20